### ISSUE TITLE
How to use ExecuteQueryMultiple cache?

### ISSUE THREAD

**Issue by seongguk92** [mikependon/RepoDb#1044]
Hello.
I am using it so well.
Is there any way to use cache for ExecuteQueryMultiple ?

~~~
var cache = CacheFactory.GetMemoryCache();
using (var connection = new SqlConnection(connectionString))
{
    var expirationInMinutes = 60 * 24; // 1 day
    var products = connection.QueryAll<Product>(cacheKey: "products",
        cacheItemExpiration: expirationInMinutes, cache: cache);
}
~~~

When proceeding with a paging operation
Call two select statements.

(paging data (5)/ total count (21))

**Comment by mikependon** [comment]
@seongguk2i the QueryMultiple is currently not supporting the caching capability, however, we are planning to introduce it on our upcoming releases. We will put this on our priority backlog.

**Comment by mikependon** [comment]
The code sample written above is for [QueryMultiple](https://repodb.net/operation/querymultiple) operation but the title is referencing to the [ExecuteQueryMultiple](https://repodb.net/operation/executequerymultiple) operation. We would like to emphasize that these are 2 separate methods and that they do different capabilities when calling both the Fluent and Raw operations. We will support the caching capability to both on the next release.

**Comment by mikependon** [comment]
The fixes to this will be available on the next version > RepoDB v1.12.10.
